agdk/game_mode/app/src/main/AndroidManifest.xml

```diff
@@ -41,13 +45,16 @@
         <category android:name="android.intent.category.LAUNCHER" />
       </intent-filter>
     </activity>
-    <!-- Enabling Game Modes -->
-    <meta-data
+
+    <!-- Enable this meta-data chunk to enable Game Mode API if you're targeting API Level 31 -->
```

Comment:
'API Level 32 or earlier' ?

Reply:
Good point, but API Level 32 is Android 12L and API Level 31 is Android 12 and we cannot go lower than API 31... Is it better to specify directly "if you're targeting API Level 31 or 32"?

Reply:
IC, please take a look a comment below.